src/billing/components/BillingPageContents.tsx

```diff
@@ -0,0 +1,25 @@
+// Libraries
+import React, {FC} from 'react'
+
+// Components
+import BillingFree from 'src/billing/components/Free/Free'
+import BillingPayAsYouGo from 'src/billing/components/PayAsYouGo/PayAsYouGo'
+import BillingCancelled from 'src/billing/components/Cancelled/Cancelled'
+
+// Utils
+import {useBilling} from 'src/billing/components/BillingPage'
+
+const BillingPageContents: FC = () => {
+  const [{account}] = useBilling()
+
+  switch (account.type) {
+    case 'free':
+      return <BillingFree />
+    case 'pay_as_you_go':
```

Comment:
nit: it would be really nice to not have any snake case in our js.  mayhaps `paygo`?

Reply:
so my rationale between keeping this as is, is the assumption that little to no API changes will be made to accommodate the UI unification work besides exposing the endpoint, so I want to stick with what the API is set to return without having to modify the returned results. Is that a safe assumption to maintain?